lib/include/jxl/codestream_header.h

```diff
@@ -314,6 +314,32 @@ typedef struct {
   /** Indicates this is the last animation frame.
    */
   JXL_BOOL is_last;
+
+  /** If coalescing is enabled, the following can be ignored; crop_x0/crop_y0 is
```

Comment:
How does this comment render in the doxygen? I'm not sure how it handles this loose comment. Maybe a short comment to each individual field affected by this can be added, like "this field is only used if coalescing is disabled"

Reply:
You can look at how this looks like in the readthedocs "check" in the PR.

Reply:
This part was modified and now no longer has a loose comment.